bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -70,7 +70,7 @@ Alternatively, you can configure the settings in the file `conf/services/openhab
 # - notification: Only push notifications are enabled, no remote access is allowed.
 # - remote: Push notifications and remote access are enabled.
 # Optional, default is 'remote'.
-#mode=
+mode=
```

Comment:
Why this change? I would guess that it was commented on purpose (like `#baseURL=` above) so it could be left untouched and defaulted.

Reply:
Thanks this was not meant to be included, I’ll revert this. I remember I changed it to see if the specified language properly did code highlighting with comments